src/adapter.js

```diff
@@ -0,0 +1,81 @@
+/* global localStorage */
+const PREFIX = 'react-entanglement-web-storage-adapter/'
+
+const key = (action, name) =>
+  `${PREFIX}${action}:${name}`
+
+const set = (action, name, data) =>
+  localStorage.setItem(key(action, name), JSON.stringify(data))
+
+const remove = (action, name) =>
+  localStorage.removeItem(key(action, name))
+
+const get = (action, name) =>
+  localStorage.getItem(key(action, name))
+
+const on = (callback) =>
+  window.addEventListener('storage', callback)
+
+const off = (callback) =>
+  window.removeEventListener('storage', callback)
+
+export default {
```

Comment:
should we make this "instanciable"? so that we can name it? `webStorageAdapter('my-app')` ?